src/som/interpreter/nodes/InstantiationNode.java

```diff
@@ -73,7 +98,8 @@ public SClass instantiateClass(final SObjectWithClass outerObj,
     public static SClass instantiate(final SObjectWithClass outerObj,
         final ClassFactory factory) {
       SClass classObj = new SClass(outerObj, instantiateMetaclassClass(factory, outerObj));
-      return singalExceptionsIfFaultFoundElseReturnClassObject(outerObj, factory, classObj);
+      return signalExceptionsIfFaultFoundElseReturnClassObject(outerObj, factory, classObj,
+          floatingNotAValueThrower, floatingCannotBeValuesThrower);
```

Comment:
Hm, can we improve this?

Ideally, the nodes should come from the caller.
The `@Specialization` above could provide instance-specific nodes, and the other callers could pass them in from static fields, if really necessary.

Reply:
Done